nni/compression/pytorch/utils/shape_dependency.py

```diff
@@ -152,8 +152,12 @@ def _get_parent_layers(self, node):
         parent_layers = []
         queue = []
         queue.append(node)
+        viewed_set = set()
```

Comment:
suggest: other variables to record visited nodes are named 'visited'

Reply:
fixed